src/components/tree-view/tree-view.tsx

```diff
@@ -34,7 +34,9 @@ export type StateMap = Map<TreeItemData, TreeItemState>;
 
 const itemIdPrefix = 'TREE_ITEM';
 
-export const TreeItem: React.SFC<TreeItemProps> = SBStateless(({ item, itemRenderer, onItemClick, stateMap, state }) => {
+const defaultItemState = { isSelected: false, isExpanded: false };
+
+export const TreeItem: React.SFC<TreeItemProps> = SBStateless(({ item, itemRenderer, onItemClick, stateMap, state = defaultItemState }) => {
```

Comment:
I wonder whether this is not a great time to introduce the lazy state initialization